robottelo/constants/__init__.py

```diff
@@ -1162,6 +1186,7 @@
     ],
     'Realm': ['view_realms', 'create_realms', 'edit_realms', 'destroy_realms'],
     'RemoteExecutionFeature': ['view_remote_execution_features', 'edit_remote_execution_features'],
+    'Report': ['create_reports'],
```

Comment:
How is this related to foreman-salt?

Reply:
I cannot answer that satisfactorily, should I remove it?

Reply:
I guess so. Maybe that permission SHOULD be there (?), but I guess that's for another PR.

Reply:
removed